src/core/lbgpu_cuda.cu

```diff
@@ -1968,7 +2344,7 @@ __device__ void calc_viscous_force(LB_nodes_gpu n_a, float *delta, float * partg
 #endif
 
   // Do the velocity interpolation
-  interpolation_two_point_coupling(n_a, position, node_index, mode, d_v, delta, interpolated_u);
+  interpolation_two_point_coupling(n_a, position, node_index, mode, d_v, delta, lees_edwards_offset, le_position, interpolated_u);
```

Comment:
same as above, this function should not depend on LE variables

Reply:
I could provide two functions (the old and the new one for LE) and distinguish their call with an #ifdef / #ifndef. But that would nearly duplicate the functions.

Reply:
No, just split functionality. Create a new function for LE and if it shares a lot of the two_point_coupling function, split that part off into a new function and call that new function from the two_point coupling and your function.